README.md

```diff
@@ -1,47 +1,172 @@
 API](https://github.com/18F/team-api-server/). These are lightweight services
 that enable a team to collect and radiate institutional knowledge and
-information. This project contains Docker components for these services, to
-enable rapid deployment of the entire suite.
-
-## Dependencies
-- Ruby >= 2.3.0
-- Docker
+information. This project contains [Docker](https://www.docker.com/)
+components for these services, to enable rapid deployment of the entire suite.
 
 ## Installation
 
-To install all the images at once: `./go build_images`
+These steps are necessary on your development machine. They are included in
+the [Deployment](#deployment) section below as well.
 
-To install specific a specific image or specific [images](#images)
+1. Install [Ruby](https://ruby-lang.org/) on your system. The [`./go`
+   script](./go) command line interface requires version 2.3.0 or greater. You
+   may wish to first install a version manager such as
+   [rbenv](https://github.com/rbenv/rbenv) to manage and install different
+   Ruby versions.
 
-```
+1. Install [Docker](https://www.docker.com/) on your system. The commands
+   encapsulated in the [`./go` script](./go) are based on version 1.10.0.
+
+1. Run `docker-machine start` to start the Docker host, followed by
+   `eval $(docker-machine env)` to configure your shell environment.
+
+1. After cloning this repository, install all the images by running `./go
+   build_images` within your copy of the repository.
+
+## Deployment
+
+1. Install your public SSH key on the remote host machine.
+
+1. Set the `REMOTE_HOST` and `REMOTE_ROOT` variables in the `./go` script as
+   necessary.
+
+1. Run `./go init_remote` if running on a brand new server. Otherwise run
+   `./go sync_remote` to bring the server up-to-date with any changes.
+
+1. Get the bundle of files containing secret data (`SECRETS_BUNDLE_FILE` in
+   the `./go` script) and run `./go push_secrets` to install them on the
+   remote host.
+
+   These files are all masked out of the repository by the `.gitignore` file.
+
+1. Run `./go ssh_remote` to log into the remote host. The working directory
+   will be the root of the repository on the remote host.
+
+1. Follow all of the steps from the [Installation](#installation) section
+   above.
+
+1. Run `./go start` to bring up all the system components, and `./go stop` to
+   stop them all.
+
+## Running locally
+
+1. Add the following to the `/etc/hosts` file of your development machine,
+   commenting out any services you're not currently attempting to emulate
+   locally:
+   ```
+   # Testing locally with 18F/knowledge-sharing-toolkit
+   # Run `docker-machine env` to get the current IP.
+   192.168.99.100 pages.18f.gov
+   192.168.99.100 pages-staging.18f.gov
+   192.168.99.100 pages-internal.18f.gov
+   192.168.99.100 pages-releases.18f.gov
+   192.168.99.100 team-api.18f.gov
+   192.168.99.100 hub.18f.gov
+   192.168.99.100 handbook.18f.gov
+   ```
```

Comment:
Won't this be a problem when you're trying to access the live sites? Not sure that we should recommend this.